eth/logmonitor.go

```diff
@@ -56,6 +58,6 @@ func (m *LogMonitor) SubscribeToJobEvents(callback func(j *Job)) {
 	m.callbacks = append(m.callbacks, callback)
 }
 
-func ParseNewJobLog(log types.Log) (transcoderAddr common.Address, broadcasterAddr common.Address, jid *big.Int) {
-	return common.BytesToAddress(log.Topics[0].Bytes()), common.BytesToAddress(log.Topics[1].Bytes()), new(big.Int).SetBytes(log.Data[0:32])
+func ParseNewJobLog(log types.Log) (transcoderAddr common.Address, broadcasterAddr common.Address, jid *big.Int, streamID string, transOptions string) {
+	return common.BytesToAddress(log.Topics[0].Bytes()), common.BytesToAddress(log.Topics[1].Bytes()), new(big.Int).SetBytes(log.Data[0:32]), string(log.Data[128:274]), string(log.Data[306:])
```

Comment:
Is the length of streamID necessarily always fixed?

Reply:
Yeah - take a look at 

https://github.com/livepeer/go-livepeer/blob/afa649cf1dc4e4717d46c09c963eb9d8e313d07f/core/streamdata.go#L36

and https://github.com/livepeer/go-livepeer/blob/afa649cf1dc4e4717d46c09c963eb9d8e313d07f/core/streamdata.go#L24

Reply:
As long as the go-livepeer client is used, the streamID length should be fixed.

Reply:
Got it. Though, if we are using the `keccak256(RENDITION_NAME)[0:4]` as the rendition identifier, the streamID length would end up being `68 (nodeID) + 32 (videoID) + 4 (rendition id) = 104` bytes long so the streamID would be at `log.Data[128:232]` right? Also, since the byte array is ABI encoded the `Data` is in 32 byte chunks so I think the length of transcodingOptions will be stored at `log.Data[256:288]` and the actual bytes for transcodingOptions will be stored at `log.Data[288:]`

Reply:
Right... We need to make that change when we switch to keccak256 for the streamID.  I'll file an issue.